duckduckgo/Android
# Add latest tosdr data

**Task/Issue URL:** https://app.asana.com/0/361428290920652/1118043392315682

**Description**:
Updates to latest tosdr data.

**Steps to test this PR**:
1. Visit yahoo.com and ensure that practices are "unknown"
1. Visit google.com and ensure that practices are "bad"

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
